src/rebar3_sbom_prv.erl

```diff
@@ -54,15 +54,34 @@ dep_info(Dep) ->
     Source = rebar_app_info:source(Dep),
     Details = rebar_app_info:app_details(Dep),
     Deps = rebar_app_info:deps(Dep),
+    Licenses0 = proplists:get_value(licenses, Details, []),
```

Comment:
Mind the added `, []`, so we can `++` with the result from `hex_metadata_licenses/1`.

Reply:
When constructing the SBoM, `[]` and `undefined` are both considered an empty value, so if no licenses are found the `licenses` attribute will still be omitted.